test/metrics-luatest/helper.lua

```diff
@@ -109,4 +109,12 @@ local function workaround_requires(path)
 end
 
 workaround_requires('test.utils')
-workaround_requires('test.psutils_linux_test_payload')
+
+require('test.utils')
```

Comment:
What is this line for?

Reply:
@DifferentialOrange Could you tell us the details about it? Thanks!

Reply:
We need to set LUA_PATH for test external server. Before the patch, LUA_PATH was needed for tarantool/metrics tests (to find metrics itself and luatest) and wasn't needed for tarantool/tarantool tests (since metrics is embedded and path for luatest is set with test-run). After this patch, some tests were migrated for using external server. Some of these migrated tests now need `test.utils` and `test.psutils_linux_test_payload` (`third_party/metrics/test/utils.lua` and `third_party/metrics/test/psutils_linux_test_payload`). Before this patch, we needed to provide  `test.utils` and `test.psutils_linux_test_payload` only to main luatest instance, which was workarounded by `workaround_requires` helper. Now we also need to provide these packages to external server. We do it here through setting LUA_PATH. You can set LUA_PATH for metrics tests through `test.utils` package `LUA_PATH` field. So this code provides `LUA_PATH` required for updates tests to be able to work in tarantool/tarantool.